rpm/zxing-cpp.spec

```diff
@@ -1,6 +1,6 @@
 Name:       zxing-cpp
 Summary:    ZXing port to C++
-Version:    1.1.1
+Version:    1.4.0
 Release:    1
 License:    ASL 2.0
 URL:        https://github.com/nu-book/zxing-cpp
```

Comment:
As per our policy should point to this git tree but not a blocker for this PR :)

Reply:
Changed.